lib/group_alert.rb

```diff
@@ -67,8 +67,8 @@ def update_post_body
       end
 
       title = topic_title
+      topic = post.topic
```

Comment:
This will error if the topic is deleted, do we already handle that somewhere else here?

Reply:
I pushed a fix for this :+1: